course/template/node/handlers/analytics.ts

```diff
@@ -1,9 +1,7 @@
 export async function analytics(ctx: Context, next: () => Promise<any>) {
-  if (ctx.method.toUpperCase() === 'GET') {
```

Comment:
I believe this change is not necessary, but I do understand that it can be a little redundant as we already define which type of HTTP method this handler will be responsible for on the `index.ts` file.